pkg/config/config.go

```diff
@@ -431,6 +438,19 @@ func toModelTags(tags []Tag) []model.Tag {
 	return ret
 }
 
+func toModelSearchTags(tags []Tag) []model.SearchTag {
+	ret := make([]model.SearchTag, 0, len(tags))
+	for _, t := range tags {
+		// regex is validated earlier
+		r, _ := regexp.Compile(t.Value)
```

Comment:
It might be worth using `regexp.MustCompile(t.Value)` instead of throwing the error away, in case there's a feature or refactor in the future that accidentally calls `toModelSearchTags()` without calling `Job.validateDiscoveryJob()`.

Reply:
Thanks for the comment! I see your point but feel a bit uncomfortable with potentially having unexpected panics in here. This mapping function already makes a lot of assumptions that validation has cleaned up a lot of data so it feels safe and the exported function ensures a consumer calls things properly.